java-symbolic-execution/src/test/files/se/Spring5Assert.java

```diff
@@ -0,0 +1,160 @@
+package symbolicexecution.behaviorcache;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.function.Supplier;
+import org.springframework.lang.Nullable;
+import org.springframework.util.Assert;
+
+class Spring5Assert {
```

Comment:
I feel the test file should not be there but on `sonar-check-test-sources`. I don't see the reason to put it there, as everything should compile. Place it in `src/main/java/symbolicexecution/behaviorcache/`